### PR TITLE
7.0 pylint

### PR DESCRIPTION
**Pull request by sbidoul**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -54,3 +54,4 @@ docs/_build/
 # Backup files
 *~
 *.swp
+*.bak
```
Comment (bwrsandman) [inline .gitignore]: How is this produced?
Changes to .gitignore should be transfered to https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.gitignore as well

Comment (sbidoul) [inline .gitignore]: It came from a 2to3 run used to transform the relative imports:

`2to3 --no-diffs -wf import *`

**Comment by pedrobaeza** [comment]
:+1:

File: .gitignore (hunk 1 above)
Comment (lepistone) [inline .gitignore]: I agree with @bwrsandman: I wouldn't change the gitignore here now.

File: account_move_template/account_document_template.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ def lines(self, line_number):
                                        self._uid,
                                        self._current_template_id,
                                        line_number)
-        if re.match('L\( *' + str(line_number) + ' *\)', line.python_code):
+        if re.match('L\\( *' + str(line_number) + ' *\\)', line.python_code):
```
Comment (lepistone) [inline account_move_template/account_document_template.py]: what about a raw string instead?

**Comment by lepistone** [comment]
other than two minor remarks, :+1: thanks.

File: .gitignore (hunk 1 above)
Comment (sbidoul) [inline .gitignore]: I removed the .bak

**Comment by sbidoul** [comment]
@pedrobaeza I integrated @lepistone's comments. All green.

**Comment by pedrobaeza** [comment]
Good! Let's fast track it then.

**Pull request merged**